numba/tests/test_sort.py

```diff
@@ -715,6 +716,79 @@ class TestQuicksortArrays(BaseQuicksortTest, TestCase):
     def array_factory(self, lst):
         return np.array(lst, dtype=np.float64)
 
+class TestQuicksortMultidimensionalArrays(BaseSortingTest, TestCase):
+
+    quicksort = make_jit_quicksort(is_np_array=True)
+    make_quicksort = staticmethod(make_jit_quicksort)
+
+    def assertSorted(self, orig, result, last_index=None):
+        # sorted() returns a list, so make sure we compare to another list
```

Comment:
This comment is probably leftover from copying previous code and can be removed.

Reply:
Removed. Thanks for pointing out.